src/explorer/Main.hs

```diff
@@ -86,6 +87,24 @@ action kad args@Args {..} transport = do
         plugins
         currentParams
         gtParams
+  where
+    lDhtWorkers
+        :: WorkMode SscGodTossing m 
+        => KademliaDHTInstance 
+        -> ([WorkerSpec m], OutSpecs)
+    lDhtWorkers kDHTInstance = dhtWorkers kDHTInstance
+
+    -- TODO: What's the point of this?
```

Comment:
The point is that we don't want to run this working while we are in recovery mode. However, this logic was moved into `dhtWorkers`. So you can remove this part.